binding.gyp

```diff
@@ -13,7 +13,7 @@
           'outputs': [
             "<(SHARED_INTERMEDIATE_DIR)/glyphs.pb.cc"
           ],
-          'action': ['protoc','-Iproto/','--cpp_out=<(SHARED_INTERMEDIATE_DIR)/','./proto/glyphs.proto']
+          'action': ['<!@(.mason/mason prefix protobuf 2.6.1)/bin/protoc','-Iproto/','--cpp_out=<(SHARED_INTERMEDIATE_DIR)/','./proto/glyphs.proto']
```

Comment:
Should `binding.gyp` be refactored somehow to run `scripts/install_mason.sh` to add `mason` and `protoc` to `$PATH`?